buildspec-release.yml

```diff
@@ -1,5 +1,10 @@
 version: 0.2
 
+env:
+  variables:
+    LC_ALL="C.UTF-8"
+    LANG="C.UTF-8"
```

Comment:
Don't these also need the ":" syntax rather than the "=" syntax? (see https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#build-spec-ref-syntax)